src/components/Input.vue

```diff
@@ -1,18 +1,43 @@
 <template>
-  <div :class="$style.addTask">
-    <input type="text" placeholder="Whats needs to be done?" name="" />
-  </div>
+  <form @submit.prevent="addTask" :class="$style.addTask">
+    <span @click="allStatusChanging"></span>
+    <input type="text" :placeholder="placeholderText" v-model="newTaskText" />
+  </form>
 </template>
 
+<script>
+import { mapMutations } from "vuex";
+export default {
+  data: () => ({
+    placeholderText: "Whats needs to be done?",
+    newTaskText: ""
+  }),
+  methods: {
+    ...mapMutations(["createTask", "changeAllStatus"]),
+    addTask() {
+      this.createTask({
+        taskText: this.newTaskText,
+        isComplete: false,
+        id: Date.now()
```

Comment:
Create Task should get only text, others params and creation should stored in mutation
Please replace Date by UUID